vendor/cookbooks/rails/templates/default/_app_nginx_location_assets.erb

```diff
@@ -0,0 +1,6 @@
+location ~ ^/(assets)/ {
+    root <%= @server_root_public_path %>;
+    gzip_static on;
+    expires max;
+    add_header Cache-Control public;
+  }
```

Comment:
yeah, I've got some issues with rendering partials, the first line is always shifted 2 spaces right